infrastructure/terraform/modules/templates-api/module_build_template_lambda.tf

```diff
@@ -0,0 +1,18 @@
+module "build_template_lambda" {
+  source = "../typescript-build-zip"
+
+  source_code_dir = "${local.lambdas_source_code_dir}/backend-api"
+  entrypoints     = [
+    var.template_api_entrypoints.create_template,
+    var.template_api_entrypoints.get_template,
+    var.template_api_entrypoints.update_template,
+    var.template_api_entrypoints.list_template,
+  ]
+}
+
+module "build_template_client" {
```

Comment:
I need this to build `backend-client`. Which `backend-api` relies on. The reason for this is explained in the `backend-client` [README.md.](https://github.com/NHSDigital/nhs-notify-web-template-management/pull/197/files#diff-ad92c6f5c10ae6ba941008f698aa7db777b5d60f691b42d2b26aebe910130b37)

Reply:
If I don't run the typescript build on the `backend-client` then I get build failures because `backend-client` has a dependency on `axios`. 

If I don't run the `typescript-build` on this package then the decencies for `backend-client` are not installed and we get a build failure stating that `axios` cannot be found.  

This is because `backend-client` is a local package rather than an external package. 

![image](https://github.com/user-attachments/assets/5e17a6fd-10b1-45de-81e8-0b216936a0b2)